libwild/src/layout.rs

```diff
@@ -3178,12 +3232,63 @@ impl<'data> EpilogueLayoutState<'data> {
             memory_offsets.increment(part_id::NOTE_GNU_BUILD_ID, build_id_sec_size);
         }
 
+        // TODO: Is it possible avoid this allocation?
```

Comment:
It's only a single allocation right? As in it's on the Epilogue, of which there's only a single instance. In that case, I definitely wouldn't worry about it.

Reply:
Yeah, it's not terrible, but I was wondering if some refactors to `VersionScript` structure could make it redundant.